rubocop/rubocop_rails.yml

```diff
@@ -1,15 +1,17 @@
 inherit_from: rubocop.yml
 
+require: rubocop-rails
+
 AllCops:
   Exclude:
     - db/schema.rb
     - 'bin/**/*'
     - 'node_modules/**/*'
     - 'vendor/**/*'
 
-Rails:
-  Enabled: true
```

Comment:
With the `require: rubocop-rails` above, we shouldn't need to enable Rails explicitly.